proxy/src/project.rs

```diff
@@ -67,13 +72,15 @@ pub struct Stats {
 }
 
 /// TODO(xla): Add documentation.
-pub async fn get(paths: &librad::paths::Paths, id: &str) -> Result<Project, error::Error> {
-    let meta = coco::get_project_meta(paths, id)?;
+pub async fn get(peer: &coco::Peer, project_urn: &str) -> Result<Project, error::Error> {
+    let meta = peer.get_project(project_urn).await?;
+    let id = meta.urn();
+    let metadata = meta.into();
 
     Ok(Project {
-        id: librad::project::ProjectId::from_str(id)?,
-        shareable_entity_identifier: format!("%{}", id),
-        metadata: meta.into(),
+        id,
+        shareable_entity_identifier: project_urn.to_string(),
```

Comment:
```suggestion
        shareable_entity_identifier: format!("%{}", project_urn)),
```
This might be why the project page broke?

Reply:
Nah it was incorrect UI behaviour see a17d58c.

Reply:
ach so, but do we still expect the `%` before the URN?